.travis.yml

```diff
@@ -2,10 +2,17 @@ sudo: false
 language: node_js
 node_js: 6
 before_install:
-  - "sh -e /etc/init.d/xvfb start"
+  - openssl aes-256-cbc -K $encrypted_f635dc9b0bb1_key -iv $encrypted_f635dc9b0bb1_iv
+    -in id_rsa_nginfinite.enc -out id_rsa_nginfinite -d
+  - sh -e /etc/init.d/xvfb start
+env:
+  global:
+  - ENCRYPTION_LABEL: f635dc9b0bb1
+  - COMMIT_AUTHOR_EMAIL: ''
 before_script:
-  - "export DISPLAY=:99.0"
+  - export DISPLAY=:99.0
```

Comment:
its already fixed can you try refreshing the page. :) https://www.dropbox.com/s/v0j14o23ydkikoc/Screenshot%202016-10-05%2015.25.28.png?dl=0